etsin_finder/frontend/js/components/qvain/datasets/dataset.jsx

```diff
@@ -189,6 +189,10 @@ const Marker = styled.div`
 `
 
 const DatasetLabel = styled(Label)`
+  color: ${({ props }) => {
+    if (props.color === 'yellow') return props.theme.color.dark
```

Comment:
`props.color` is undefined here and will cause a crash, 

Suggestion: Change row 118 to
`          <Translate color="dark" content="qvain.datasets.oldVersion" component={DatasetLabel} />
`

Reply:
Did some small changes and added check that color is defined, defaulting to white. App crash resolved.